bedrock/firefox/templates/firefox/whatsnew/base.html

```diff
@@ -28,7 +28,5 @@
 {% endblock %}
 
 {% block site_footer %}
-  {% with theme_class='mzp-t-light' %}
-    {% include 'includes/protocol/footer/footer.html' %}
-  {% endwith %}
+  {% include 'includes/protocol/footer/footer.html' %}
 {% endblock %}
```

Comment:
**Nit**: can this whole block be removed now since it's no different to the base template?